src/indra_cogex/apps/templates/base_form.html

```diff
@@ -1,13 +1,37 @@
 {% extends "base.html" %}
-
 {% import "bootstrap4/form.html" as wtf %}
-
 {% block container %}
     <div class="card card-body bg-light">
         <h1 class="display-4">{% block header %}{% endblock %}</h1>
         <p class="lead">
             {% block lead %}{% endblock %}
         </p>
-        {{ wtf.render_form(form) }}
+        <form id="main-form" method="POST">
+            {{ wtf.render_form(form) }}
+        </form>
```

Comment:
I'm not sure you need to add this, the form tag is autogenerated by `wtf.render_form` and you can always select the form element using [class name](https://www.w3schools.com/jsref/met_document_getelementsbyclassname.asp) or by [tag name](https://www.w3schools.com/jsref/met_document_getelementsbytagname.asp).
Another option is to look into if you can add the `id` as an extra parameter to `wtf.render_form`.

Reply:
Fixed in 73558979